Packs/DeveloperTools/Scripts/GetServerInfo/GetServerInfo.yml

```diff
@@ -14,6 +14,6 @@ timeout: '0'
 type: python
 subtype: python3
 dockerimage: demisto/python3:3.10.14.92207
-fromversion: 6.10.0
+fromversion: 6.9.0
```

Comment:
What is the reasoning behind this?

Reply:
this is a script i added for this test playbook, then @thefrieddan1 refactored and moved to developer tools. We still run tests on 6.9, so my tpb failed there because this scriot wasnt available

Reply:
👍